### ISSUE TITLE
Add link to marketplace in dev menu

### ISSUE THREAD

**Issue by GonzaloTorreras** [williamspiro/HubSpot-Developer-Extension#104]
**Is your feature request related to a problem? Please describe.**
Specially for new portals, I want to access to the Marketplace and I always forget where I can find in the new menu.

**Describe the solution you'd like**
Would be nice add the link to the marketplace in to the dev menu too.

**Additional context**
Lol, not sure if this is the place to put that xD

**Comment by TheWebTech** [comment]
I agree the extension does need to provide quick access to the marketplace.

I had originally thought about putting it in the develop tab of the popup but it may make more sense in the dev menu.

I don't personally access it alot - so I'd maybe add it to the bottom?

If it wasn't obvious, I tried to organize the links in the dev nav based on how frequently each would be used. (not alphabetically)

Would you agree with that?

**Comment by GonzaloTorreras** [comment]
Sure, at the bottom is good for me
